versions/versions.gradle

```diff
@@ -7,27 +7,27 @@
 ext.CONFIG.versions = [
     // Java compatibility target version
     java      : '1.7',
-    kotlin    : '1.2.0',
+    kotlin    : '1.3.21',
 
     // Android plugins, tools, SDK and libraries
     android   : [
         libraries: [
-            playServices: '11.0.4',
-            support     : '26.0.0',
+            playServices: '16.0.0',
+            support     : '28.0.0',
             volley      : '1.1.1',
         ],
-        plugin: '3.3.1',
+        plugin: '3.3.2',
         sdk      : [
             compile: 28,
             target : 28,
-            min    : 15,
+            min    : 21,
         ],
     ],
 
     // Third party dependencies of our SDK
     thirdParty: [
-        autoParcelGson: '0.2',
-        gson          : '2.8.1',
-        threetenbp    : '1.3.6',
+        autoParcelGson: '1.0.3',
+        gson          : '2.8.5',
+        threetenbp    : '1.3.8',
```

Comment:
@corycaywood should we remove these? i don't think we need a central registry of these dependencies that are used sometimes and adhere to semantic versioning.

Reply:
Yeah, I think we can remove them. We also don't use them in many SDKs anymore.